src/routes/+page.svelte

```diff
@@ -1,110 +1,116 @@
 <script>
   import { base } from "$app/paths"
 
-  import Book from "svelte-bootstrap-icons/lib/Book.svelte"
-  import Play from "svelte-bootstrap-icons/lib/Play.svelte"
-  import ButtonLinks from "$lib/components/ButtonLinks.svelte"
+  import Button from "lib//components/Button.svelte"
+  import Link45deg from "svelte-bootstrap-icons/lib/Link45deg.svelte"
 
-  import { github_url } from "$lib/constants"
+  $: heroBgImage = `background-image: url("${base}/images/topography.svg");`
 </script>
 
-<h1 class="font-bold text-5xl">Climate Town</h1>
-<h2 class="font-semibold text-4xl pl-16">Knowledge Hub</h2>
-
-<p class="pt-16 leading-7">
-  Welcome to the Climate Town Knowledge Hub, your one-stop shop for important
-  resources in the fight against climate change! This website contains a
-  collection of online resources, initiatives, and tools suggested by the
-  Climate Town Discord community.
-</p>
-
-<p class="pt-6 pb-3 leading-7">
-  Some of the resource tags in the Knowledge Hub are special. These are prefaced
-  with emojis:
-</p>
-<ul class="list-disc leading-6">
-  <li class="mb-2">
-    <span class="font-semibold">📚 Knowledge Hubs:</span> These are initiatives similar
-    to the Climate Town Knowledge Hub. These initiatives curate various lists of
-    climate-related resources, lists which may even be more specific in topic than
-    the list of resources provided here, taking full advantage of the network effect!
-  </li>
-  <li class="mb-2">
-    <span class="font-semibold">🍞 Rollie's Choice:</span> These are resources endorsed
-    by the Climate Town team.
-  </li>
-  <li class="mb-2">
-    <span class="font-semibold">✨ Mod Choice:</span> These are resources endorsed
-    by the moderators of the Climate Town Discord.
-  </li>
-</ul>
-
-<div class="grid grid-cols-2 pb-10">
+<div
+  class="heroBanner relative flex justify-center items-center flex-col"
+  style={heroBgImage}
+>
   <div
-    class="rounded-lg shadow-xl border-b-4 border-r-4 m-5 p-3 border-zinc-200 dark:border-zinc-900 md:col-span-1 col-span-2 flex flex-col"
+    class="heroBanner_overlay absolute w-full flex flex-col justify-center text-center bg-gradient-to-b from-transparent to-white dark:from-transparent dark:to-neutral-900 h-full"
   >
-    <h3>Resources</h3>
-    <div class="my-3 text-zinc-600 dark:text-zinc-300 grow">
-      A crowdsourced collection of a variety of climate resources. Want to
-      suggest a resource? Please <a
-        href="{github_url}/issues/new/choose"
-        class="underline">fill out our form</a
-      >
-      or
-      <a href={github_url} class="underline">edit on GitHub directly</a>
+    <div class="max-w-7xl mx-auto">
+      <h1 class="font-bold text-5xl mx-6">
```

Comment:
design wise on mobile, looks like the headings should shrink some for that context
<img width="491" alt="Screen Shot 2024-08-21 at 4 15 51 PM" src="https://github.com/user-attachments/assets/2847063c-b8fb-4ba9-858e-03b9ab23c918">